Library/Formula/multimarkdown.rb

```diff
@@ -3,7 +3,7 @@
 class Multimarkdown < Formula
   homepage 'http://fletcherpenney.net/multimarkdown/'
   # Use git tag instead of the tarball to get submodules
-  url 'https://github.com/fletcher/MultiMarkdown-4.git', :tag => '4.6'
+  url 'https://github.com/fletcher/MultiMarkdown-4.git', :tag => '4.7.1'
```

Comment:
Could you explain more or point me to the right docs, I am not sure if I know what to do. I found out that there are two types of `revision`. I can add it after `head` like `revision 1` which I guess force recompile. Or use `:revision` on url which I don't what it does.

Thanks,
Amir.